nbs/07_clean.ipynb

```diff
@@ -168,6 +174,87 @@
     "               'source': 'awesome_code'})"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tst2 = {'cell_type': 'code',\n",
```

Comment:
I'd suggesting simplifying this down to just the absolute minimum needed to test. Having lots of code in tests makes them hard to read and understand, and harder to maintain.

Reply:
By moving this into `nbdev_clean_nbs` directly this test is no longer needed